### PR TITLE
Fix #68

### LINKED ISSUE #68: Feature request: Set minimum brightness
I think it would be good to set a minimum brightness to which one can reduce. I am not sure if this is the same as this [issue](https://github.com/lordamit/Brightness/issues/45).

It wouldn't be nice to loose sight of your desktop even if you have two screen enabled. Accidents could happen like switching to one screen only whose brightness is set to zero.

### PR DESCRIPTION
**Pull request by apandada1**
Something like this can be implemented in v2 too.

In the new version (to be packaged) brightness will return to 100% in primary monitor if the command `brightness-reset` is run

### CODE REVIEW

File: src/usr/bin/brightness-reset [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def __init__(self):
+        self.detected_devices = self.detect_display_devices()
+	self.primary_name = self.detected_devices[0]
+	cmd1 = "xrandr --output \
+                %s --brightness %s" % (self.primary_name, 1)
+	system(cmd1)
+
+
+if __name__ == '__main__':
+    app = wx.App()
```
Comment (lordamit) [inline src/usr/bin/brightness-reset]: why is this required?

File: src/usr/bin/brightness-reset [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import wx
```
Comment (lordamit) [inline src/usr/bin/brightness-reset]: why this?

File: src/usr/bin/brightness-reset [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            words = line.split(' ')
+            for word in words:
+                if word == 'connected':
+                    connected_devs.append(words[0])
+        return connected_devs
+
+    def __init__(self):
+        self.detected_devices = self.detect_display_devices()
+	self.primary_name = self.detected_devices[0]
+	cmd1 = "xrandr --output \
```
Comment (lordamit) [inline src/usr/bin/brightness-reset]: why is this cmd1? why not command?

File: src/usr/bin/brightness-reset [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+                %s --brightness %s" % (self.primary_name, 1)
+	system(cmd1)
+
+
+if __name__ == '__main__':
+    app = wx.App()
+    BrightnessRestore()
+    app.MainLoop()
```
Comment (lordamit) [inline src/usr/bin/brightness-reset]: what does this do?

**Comment by lordamit** [comment]
Check the notes :ghost: 

Also, we really do not need dependency on wx for resetting brightness. That's overkill.

File: src/usr/bin/brightness-reset (hunk 3 above)
Comment (apandada1) [inline src/usr/bin/brightness-reset]: At first I tried to restore brightness of both displays (cmd1 &cmd2), but it showed some errors when no secondary display was connected. I forgot to remove 1 from cmd1.

File: src/usr/bin/brightness-reset (hunk 4 above)
Comment (apandada1) [inline src/usr/bin/brightness-reset]: I actually had to copy code from brightness controller as the program did not work without class BrightnessRestore(wx.Frame): and if __name__ == '__main__': sections. Any idea why? However line 31 and line 33 are not required.

File: src/usr/bin/brightness-reset [hunk 5]
```diff
@@ -28,6 +27,4 @@ class BrightnessRestore(wx.Frame):
 
 
 if __name__ == '__main__':
```
Comment (apandada1) [inline src/usr/bin/brightness-reset]: @lordamit Why is this line required?

File: src/usr/bin/brightness-reset (hunk 3 above)
Comment (lordamit) [inline src/usr/bin/brightness-reset]: Anyone can code, but it is hard to write beautiful, understandable and maintainable code :wink:
